app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -383,7 +393,7 @@ class BrowserFragment :
     }
 
     private fun initialiseCustomTabUi(view: View) {
-        val customTabConfig = session.customTabConfig!!
+        val customTabConfig = tab.ifCustomTab()!!.config
```

Comment:
```suggestion
        val customTabConfig: CustomTabConfig = tab.ifCustomTab()?.config ?: return
```

nit: early return, maybe?

Reply:
Yeah, this whole code is not nice. At this point we already know that it's a custom tab, so having to deal with nullability here is annoying. Hm, actually, maybe I can just pass in the `CustomTabConfig` if the check happens outside, I'll try that and update the PR.

Reply:
It shows up now, so I fixed the conflict in the PR and we can merge it.